### PR TITLE
completed css-two-column-layout exercise

### PR DESCRIPTION
**Pull request by AnthonyUrbina**
https://user-images.githubusercontent.com/109625982/188284843-f07f9d30-8bfc-4e32-8cb2-4897d2780994.mov

### CODE REVIEW

File: css-two-column-layout/styles.css [hunk 1]
```diff
@@ -0,0 +1,82 @@
...
+}
+
+.container {
+  margin: 0 auto;
+  width: 100%;
+}
+
+.row-2 {
+  display: flex;
+}
```
Comment (codingcodymiller) [inline css-two-column-layout/styles.css:41]: What happened to `.row`? You should never make numbered classes, it isn't descriptive of their purpose at all.

File: css-two-column-layout/styles.css [hunk 2]
```diff
@@ -0,0 +1,82 @@
...
+
+img {
+  padding-bottom: 4.5rem;
+  padding-right: 8px;
+}
+
+.row-3 {
+  text-align: center;
+}
```
Comment (codingcodymiller) [inline css-two-column-layout/styles.css:68]: This would be better named as `.text-center` or something to that effect

**Review by codingcodymiller — CHANGES_REQUESTED** [review]

**Comment by AnthonyUrbina** [comment]
got it
just fixed. i had done that to center rows but idk why i did it that way since i couldve used justify content

File: css-two-column-layout/styles.css [hunk 3]
```diff
@@ -0,0 +1,82 @@
...
+
+h3 {
+  text-transform: uppercase;
+  letter-spacing: 0.1rem;
+}
+
+.header-column-full {
+  text-align: center;
+}
```
Comment (codingcodymiller) [inline css-two-column-layout/styles.css:32]: Again, this should be named `text-center` or something to that effect. `column-` classes only include `flex-basis` or `width`, anything else is not styling to apply to a `column-` class

**Review by codingcodymiller — CHANGES_REQUESTED** [review]

**Review by codingcodymiller — APPROVED** [review]

**Pull request merged**